setup.py

```diff
@@ -9,7 +9,9 @@
     license='',
     install_requires=['thrift==0.9.2',
                       'opentracing==1.0rc3',
-                      'jsonpickle'],
+                      'jsonpickle',
+                      'basictracer==1.0rc1',
+                      'protobuf>=3.0.0b2.post2'],
```

Comment:
Gotcha I think it's the latter too. So if I delete that, would this mean that the users would have to either put the basictracer library in the same directory themselves or otherwise download it?